meteor/client/ui/Settings/components/ConfigManifestOAuthFlow.tsx

```diff
@@ -134,18 +172,22 @@ export const ConfigManifestOAuthFlowComponent = withTranslation()(
 								</label>
 							) : (
 								<label className="field">
-									{t('Access token')}
-									<div className="mdi">
-										<div>{t('Click on the link below and accept the permissions request.')}</div>
-										<div>
+									<div>
+										<div>{t('Click on the button below and accept the permissions request.')}</div>
+										<div className="mvs">
 											{device.accessTokenUrl ? (
 												<a href={device.accessTokenUrl} target="_blank" rel="noreferrer">
-													{device.accessTokenUrl}
+													<button className="btn btn-primary">{t('Authorize App Access')}</button>
```

Comment:
A `button` in an `<a href>` doesn't seem right. I think this should work OK with just having an `<a className='btn'>`. If not - `window.open`?